x-pack/solutions/observability/plugins/slo/public/hooks/use_enable_slo.ts

```diff
@@ -44,6 +44,15 @@ export function useEnableSlo() {
       onSuccess: (_data, { name }) => {
         queryClient.invalidateQueries({ queryKey: sloKeys.lists(), exact: false });
         queryClient.invalidateQueries({ queryKey: sloKeys.details(), exact: false });
+        queryClient.invalidateQueries({
+          queryKey: sloKeys.definitions({
+            search: name,
+            page: 1,
+            perPage: 10,
+            includeOutdatedOnly: false,
+          }),
+          exact: false,
```

Comment:
no need for the params, just use whatever function that yields ['slo', 'definitions']